README.org

```diff
@@ -20,6 +36,19 @@
       to attempt to move these spec changes into the official matrix-org spec.
   - Provide a central location for other important documentation on the Matrix
     ecosystem.
+
+* Repositories
```

Comment:
Seems to be missing `eventually-doc` entirely?

Reply:
I don't have a clear view of that purpose. I would add it tomorrow after a talk with @non-Jedi

Reply:
eventually-doc may need a rename depending on what we decide. Can see
several paths forward:

1. Maintain all docs specific to Eventually Matrix in eventually-doc
2. Split docs related to Eventually Matrix effort into a repo with efforts
   related to documenting current implementations and another repo with more
   speculative contents.
3. Just go with option 1 for now with intention to evaluate and change once
   we've tested workflow better.
   
My preference would be for 1 or 3, but I can see argument for 2.

Reply:
My vote is on 3 for the time being, but I expect that when the time comes to evaluate, I will be arguing for something very much like 2

Reply:
3 for me as well. I wouldn't do more work than necessary for now. Let's see if this gains any momentum in the first place.